raiden/constants.py

```diff
@@ -36,6 +36,7 @@
 EMPTY_HASH_KECCAK = keccak(EMPTY_HASH)
 EMPTY_SIGNATURE = bytes(65)
 EMPTY_MERKLE_ROOT = Locksroot(bytes(32))
+EMPTY_SECRET = b''
```

Comment:
Using `None` is better then `b''` since that represent the absence of a value, instead of the empty value.

Reply:
Initially I made EMPTY_SECRET to be None. 
This however causes crashes in multiple places.. 

For example 
https://github.com/raiden-network/raiden/blob/ad8ae3c976090b6437ea43a477ff3234b0fab90a/raiden/raiden_service.py#L690

and 

https://github.com/raiden-network/raiden/blob/ad8ae3c976090b6437ea43a477ff3234b0fab90a/raiden/storage/wal.py#L76

And I think some other places. 

Please advise @hackaugusto

Reply:
Yeah, IMO we should fix this.

Using `b''` doesn't raise exceptions because it's the same datatype as the secret, calls `sha3(b'')` work but `sha3(None)` raise. The problem here is that we may have hidden bugs if we just use `b''` to avoid raising exceptions.